### PR TITLE
Allow explicit empty config values for intervals.

### LINKED ISSUE #1865: Default global.rc file gives validation error
I got a new workstation, so I'm setting it up with the latest cylc version (6.10.1-37-g2c9bb), and I'm now getting validation errors with the global.rc file.  I cleared out the old and ran:

`cylc get-site-config > $HOME/.cylc/global.rc`

to generate the new configuration file.  Then, I tried validating a suite and got:

```
[whitcomb@whitcomb-dev ~]$ cylc validate -v /ftp/receive/whitcomb/suite.rc
Loading site/user config files
Reading file /users/whitcomb/.cylc/global.rc
WARNING: ignoring bad site config /users/whitcomb/.cylc/global.rc:
Illegal ISO 8601 interval value: [task events][execution timeout]
```
it's also not suite-specific:
```
[whitcomb@whitcomb-dev ~]$ cylc version
WARNING: ignoring bad site config /users/whitcomb/.cylc/global.rc:
Illegal ISO 8601 interval value: [task events][execution timeout]
6.10.1-37-g2c9bb
```

If I look at the file, it shows that execution timeout entry as blank.  I haven't changed anything, but it appears that the validation code is looking for something that's not present in the internal configuration.

Expected result is to not get a bad site config by using the software's method to dump the current configuration from scratch - the actual result is a validation error with a pristine `global.rc` file.

### PR DESCRIPTION
**Pull request by hjoliver**
Close #1865 

@arjclark - can you review & reassign (Matt says he's away for the rest of the week).

### CODE REVIEW

**Comment by arjclark** [comment]
Will do

File: lib/cylc/cfgspec/globalcfg.py [hunk 1]
```diff
@@ -32,7 +32,6 @@
 from parsec.util import itemstr
 from parsec.upgrade import upgrader, converter
 from parsec.fileparse import parse
-from isodatetime.data import Calendar
```
Comment (arjclark) [inline lib/cylc/cfgspec/globalcfg.py]: Guess this is just tidying an unused import?

**Comment by arjclark** [comment]
Looks OK to me. No problems from the test-battery in my environment.

@benfitzpatrick - please review 2 in Matt's absence.

File: lib/cylc/cfgspec/globalcfg.py (hunk 1 above)
Comment (hjoliver) [inline lib/cylc/cfgspec/globalcfg.py]: yes

**Pull request merged**